src/bio/bio_xstream.c

```diff
@@ -897,15 +898,15 @@ bio_xsctxt_alloc(struct bio_xs_context **pctxt, int tgt_id)
 
 		rc = spdk_conf_read(config, nvme_glb.bd_nvme_conf);
 		if (rc != 0) {
-			D_ERROR("failed to read %s, rc:%d\n",
-				nvme_glb.bd_nvme_conf, rc);
+			D_ERROR("failed to read %s, "DF_RC"\n",
+				nvme_glb.bd_nvme_conf, DP_RC(rc));
 			rc = -DER_INVAL; /* spdk_conf_read() returns -1 */
 			goto out;
 		}
 
 		if (spdk_conf_first_section(config) == NULL) {
-			D_ERROR("invalid format %s, rc:%d\n",
-				nvme_glb.bd_nvme_conf, rc);
+			D_ERROR("invalid format %s, "DF_RC"\n",
+				nvme_glb.bd_nvme_conf, DP_RC(rc));
```

Comment:
In general, we should never do this when printing the return value of a 3rd party API

Reply:
moved all D_ERROR to after rc is set to DER.